src/views/privacypolicy/l10n.json

```diff
@@ -0,0 +1,34 @@
+  "privacypolicy.thirdparty": "We may disclose some of the information we collect to third-party service providers that help us manage communications to and from the Scratch website and improve website performance. We are satisfied that these service providers have privacy policies that restrict them from further disclosing any of your information.",
+  "privacypolicy.exceptions": "Other than as described above, we will never share personally identifiable information about you with any other person, company, or organization, except:",
+  "privacypolicy.lawrequirement": "As required to comply with our obligations under the law.",
+  "privacypolicy.technicalreasons": "For technical reasons, if we are required to transfer the data on our servers to another location or organization.",
+  "privacypolicy.updateheader": "How can I update my personal information?",
+  "privacypolicy.updateuser": "You can update your password, email address, and country through the <a href=\"/account/password_change\">Account Settings</a> page. You can also reset your password through the <a href=\"/account/password_reset\">Account Reset</a> page. You cannot change your username, but you can make a new account and manually copy your projects to the new account.",
+  "privacypolicy.deleteuser": "If you want to delete your account entirely, log in to Scratch, and then click your username in the top right-hand corner. Select “Account Settings,” then click the “I want to delete my account” link at the bottom of the page.",
+  "privacypolicy.infoprotectionheader": "How does the Scratch Team protect my information?",
+  "privacypolicy.infoprotectionbody": "The Scratch Team has in place physical and electronic procedures to protect the information we collect on the Scratch website. We strictly limit individual access to the Scratch servers and the data we store on them. However, as effective as these measures are, no security system is impenetrable. We cannot completely guarantee the security of our database, nor can we guarantee that the information you supply will not be intercepted while being transmitted to us over the Internet.",
+  "privacypolicy.policychangesheader": "Notifications of Changes to the Privacy Policy",
+  "privacypolicy.policychangesbody": "We review our security measures and Privacy Policy on a periodic basis, and we may modify our policies as appropriate. We may also change or update our Privacy Policy if we add new services or features. If we make any changes to our privacy practices, we will amend this Privacy Policy accordingly and post the amended policy on the Scratch website. We encourage you to review our Privacy Policy on a regular basis.",
```

Comment:
I think the issue with travis is this trailing comma. In JSON, the final element may not end with a comma.